### PR TITLE
speed up unit tests by switching to fetch_parameters_from_easyconfig

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

**Comment by boegel** [comment]
Jenkins: test this please

File: test/easyconfigs/suite.py [hunk 1]
```diff
@@ -45,6 +45,9 @@
 fancylogger.disableDefaultHandlers()
 fancylogger.setLogLevelError()
 
+# make sure no deprecated behaviour is triggered
```
Comment (wpoely86) [inline test/easyconfigs/suite.py]: wait what?

Comment (boegel) [inline test/easyconfigs/suite.py]: yes, the tests should never trigger deprecated (or no longer supported) behaviour...

Comment (wpoely86) [inline test/easyconfigs/suite.py]: a yeah, I was confused. This means sure it will always fail.

Comment (boegel) [inline test/easyconfigs/suite.py]: well, only if deprecated behaviour is triggered :)

**Comment by boegel** [comment]
going in, thanks for the review @wpoely86!

**Pull request merged**